package.json

```diff
@@ -25,11 +25,9 @@
     "bootstrap": "~3.4.0",
     "bootstrap5": "npm:bootstrap@^5.1.3",
     "font-awesome": "~4.7.0",
-    "getusermedia-js": "~1.0.0",
```

Comment:
Ah, if we remove these here, won't it break v1? Shall we leave them in even though v2 won't be using them?

Reply:
Or alternatively, should we implement the same changes in index2.html into index.html and fix both v1 and v2 in this respect?

Reply:
OH sorry

Reply:
Let's not break v1 for now. We can break it after v2 is complete , when @stephaniequintana and I have a synchronized codebase.

Reply:
@jywarren , package.json has been fixed. Please may I merge?